geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -729,6 +762,17 @@ public LocatorState start() {
               bindAddress, true, getDistributedSystemProperties(),
               getHostnameForClients(),
               Paths.get(workingDirectory));
+          if (isJavaVersionAtLeast(JAVA_1_8) && isJavaVersionAtMost(JAVA_1_8)
```

Comment:
shouldn't this use the new "isJavaVersion" method you added?

Reply:
We have removed isJavaVersion. And keep isJavaVersionAtLeast and isJavaVersionAtMost, which are more convenient to handle multiple Java versions, instead of a specific version.